Source/animationinfos.cpp

```diff
@@ -48,4 +48,78 @@ int AnimationInfos::GetFrameToUseForRendering()
 	return absoluteAnimationFrame;
 }
 
+void AnimationInfos::SetNewAnimation(uint8_t *Peq, int numFrames, int Delay, AnimationDistributionParams params /*= AnimationDistributionParams::None*/, int numSkippedFrames /*= 0*/, int distributeFramesBeforeFrame /*= 0*/)
+{
+	pData = Peq;
+	FrameLen = numFrames;
```

Comment:
This is a very jarring name clash. When extracting the function, we should rename the arguments appropriately imho.

Reply:
Fixed 🙂